ansible/roles/dev-desktop/tasks/dependencies.yml

```diff
@@ -59,6 +59,12 @@
       - zsh
     state: present
 
+# we don't need it because we don't need to send emails
+- name: Uninstall postfix
+  apt:
+    name: postfix
+    state: absent
```

Comment:
I removed postfix because there was an alert related to it.
Instead of fixing it I decided to remove this package as we don't need it imo.